docs/source/release/v6.3.0/mantidworkbench.rst

```diff
@@ -10,5 +10,7 @@ New and Improved
 
 Bugfixes
 --------
+- Fixed an issue to set up "Start Time" in `StartLiveDataDialog`.
```

Comment:
Can we just tweak this to be more clear, something like

- Fixed arbitrary values not being accepted as the "Start Time" in `StartLiveDataDialog`

Reply:
Thanks!  It is applied.